src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java

```diff
@@ -0,0 +1,83 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mobile;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.Assertion;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.nio.channels.ByteChannel;
+
+/**
+ * Created by rcunningham on 6/30/15.
+ */
+public class MobileSignupPageObject extends MobileBasePageObject {
+
+    public MobileSignupPageObject(WebDriver driver) { super(driver); }
+
+    // UI Mapping
+    @FindBy(css = "#signupEmail")
+    private WebElement signupEmail;
+    @FindBy(css = "#signupUsername")
+    private WebElement signupUsername;
+    @FindBy(css = "#signupPassword")
+    private WebElement signupPassword;
+    @FindBy(css = "#signupBirthDate")
+    private WebElement signupBirthdate;
+    @FindBy(css = ".birth-month")
+    private WebElement signupBirthMonth;
+    @FindBy(css = ".birth-day")
+    private WebElement signupBirthDay;
+    @FindBy(css = ".birth-year")
+    private WebElement signupBirthYear;
+    @FindBy(css = "#signupSubmit")
+    private WebElement signupButton;
+    @FindBy(css = ".avatar")
+    private WebElement avatar;
+
+
+
+    public void typeEmailAddress(String email) {
+        waitForElementByElement(signupEmail);
+        signupEmail.sendKeys(email);
+    }
+
+    public void typeUsername(String username) {
+        waitForElementByElement(signupUsername);
+        signupUsername.sendKeys(username);
+    }
+
+    public void typePassword(String password) {
+        waitForElementByElement(signupPassword);
+        signupPassword.sendKeys(password);
+    }
+
+    public void typeBirthdate(String month, String day, String year) {
```

Comment:
I need to write other tests that will give invalid values for each of these items, which is why I created them each separately.  Do you have a better solution?